src/plugins/kibana_usage_collection/server/collectors/event_loop_delays/event_loop_delays_usage_collector.test.ts

```diff
@@ -36,7 +37,8 @@ describe('registerEventLoopDelaysCollector', () => {
       logger,
       usageCollectionMock,
       mockRegisterType,
-      mockGetSavedObjectsClient
+      mockGetSavedObjectsClient,
+      new Observable<void>()
```

Comment:
Probably a NIT, but shouldn't we then close the observable during the test's teardown?

Reply:
Touché! I've updated the code to use a Subject and `complete()` it properly.